### PR TITLE
Add puppeteer screenshot example

### LINKED ISSUE #16: Puppeteer example
Add an example running puppeteer to test a website and output if tests passed in an HTML or the errors.

Would be a good reference on how to run puppeteer inside Docker with Now.

### PR DESCRIPTION
**Pull request by pranaygp**
Closes #16 

tl;dr A demo of how to fit chrome inside a 100MB docker image for use with puppeteer. Try it out at https://misc-screenshot.now.sh

This is a demo of how to take a screenshot using puppeteer on Now. The built image fits the 100MB image size constraint by compressing `chromium` and it's associated lib files using `zstd` in [dicitionary mode](https://github.com/facebook/zstd#dictionary-compression-how-to). It also uses the `--ultra` feature flag to enable a higher level `22` compression which gives us a compression ratio of 3.5x instead of 2.9x to further make it fit (this makes compression slower, but decompression seems to be just as fast so that's good)

### CODE REVIEW

**Comment by pranaygp** [comment]
This is at 89.6MB right now which means I might be able to switch back to `lz4` instead of `zstd` to make decompression and bootup faster. I might implement that or just create an issue for it for later 😄

**Comment by shawnmaten** [comment]
@pranaygp I wanted to ask about this compression technique you've used to get the image size down. Is this something unique you came up with for this, or have you seen this used elsewhere? It's new to me and I was wondering if I might find other examples or discussion of it. I'm curious about the implications of it. It seems like a clever idea... but also seems hacky and potentially bad.

**Comment by pranaygp** [comment]
@shawnmaten This example pushes the limits on what's possible in a 100MB image and serves as a proof of concept that even a rather expensive set of operations (i.e. booting up a vm, extracting a zstd archive to "install" chrome, booting up chrome, navigating to a webpage and taking a screenshot) could happen in less than 5s with Now.

Without divulging too much information, I can say that the extra cost here (mostly, the decompression step) will soon be absorbed by Now. Once that's implemented, the entire process will run in less than a second.

In addition, we're also working on increasing that 100MB limit so you don't have to go through these unnecessary extra steps of compression.
